.github/workflows/docs.yml

```diff
@@ -0,0 +1,18 @@
+name: Markdown Tests
+on:
+  push:
+    branches:
+      - main
```

Comment:
This workflow will not run on PR (and we need this linter to reduce review for PRs).

Copy this block from `test.yml` https://github.com/hplush/slowreader/blob/main/.github/workflows/test.yml#L10